pepys_import/database/default_data/Platforms.csv

```diff
@@ -1,3 +1,3 @@
-name,nationality,type,privacy
-PLATFORM-1,UNITED KINGDOM,TYPE-1,PRIVACY-1
-PLATFORM-2,TURKEY,TYPE-2,PRIVACY-2
\ No newline at end of file
+name,pennant,nationality,type,privacy
```

Comment:
Baris, could you remove the pennant number from these two CSV files, please?

Reply:
Sure. Pennant columns deleted:
https://github.com/debrief/pepys-import/pull/51/commits/764b9c43e9157100431f6154dbeca5e10729432a